cmake/sdlfind.cmake

```
@@ -0,0 +1,9 @@
+
+macro(sdlFindALSA)
+    find_package(ALSA MODULE)
+    if(ALSA_FOUND AND (NOT TARGET ALSA::ALSA) )
+        add_Library(ALSA::ALSA UNKNOWN IMPORTED)
+        set_property(TARGET ALSA::ALSA PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${ALSA_INCLUDE_DIRS})
+        set_property(TARGET ALSA::ALSA APPEND PROPERTY IMPORTED_LOCATION ${ALSA_LIBRARIES})
```

I would use `ALSA_INCLUDE_DIR` and `ALSA_LIBRARY`.
This is the [same as upstream](https://github.com/Kitware/CMake/blob/e0a35098bb2874ac7d6bca5b1a989c293ad18922/Modules/FindALSA.cmake#L70-L72).
```suggestion
        add_Library(ALSA::ALSA UNKNOWN IMPORTED)
        set_property(TARGET ALSA::ALSA PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${ALSA_INCLUDE_DIR})
        set_property(TARGET ALSA::ALSA APPEND PROPERTY IMPORTED_LOCATION ${ALSA_LIBRARY})
```